src/web/layouts/Standard.stories.tsx

```diff
@@ -95,6 +95,11 @@ export const CommentStory = (): React.ReactNode => {
 	return <HydratedLayout ServerCAPI={ServerCAPI} />;
 };
 CommentStory.story = { name: 'Comment' };
+CommentStory.parameters = {
+	// Cutout/byline interaction is a specific issue
+	// we look out for on mobile on opinion content
+	chromatic: { viewports: [320, 1300] },
```

Comment:
Maybe we can import these breakpoints from Source?

`import { breakpoints } from '@guardian/src-foundations/mq';`

And then (adjust as required)

```
		chromatic: {
			viewports: [
				breakpoints.mobile,
				breakpoints.mobileMedium,
				breakpoints.phablet,
				breakpoints.tablet,
				breakpoints.desktop,
				breakpoints.leftCol,
				breakpoints.wide,
			],
		},
```

Reply:
Yes, good shout 👍